src/governance/ProfitManager.sol

```diff
@@ -60,6 +60,13 @@ contract ProfitManager is CoreRef {
     /// if the loss is greater than the surplus buffer, the `creditMultiplier` is updated down.
     uint256 public surplusBuffer;
 
+    /// @notice amount of first-loss capital for a given term.
+    /// This is a number of CREDIT token held on this contract that can be used to absorb losses in
+    /// cases where a loss is reported through `notifyPnL` in a given term.
+    /// When a loss is reported in a given term, its termSuplusBuffer is donated to the general
+    /// surplusBuffer before calculating the loss.
+    mapping(address => uint256) public termSurplusBuffer;
```

Comment:
what happens when an individual term has a loss greater than the current surplus buffer, but the sum of all term surplus buffers would exceed that of the loss on the term?

it seems that in this case, CREDIT price would take a hit even though enough surplus buffer capital exists to cover the loss.